community/cypher/frontend-3.3/src/main/scala/org/neo4j/cypher/internal/frontend/v3_3/parser/Literals.scala

```diff
@@ -30,6 +30,33 @@ trait Literals extends Parser
   def Variable: Rule1[ast.Variable] =
     rule("a variable") { SymbolicNameString ~~>> (ast.Variable(_) ) }.memoMismatches
 
+  def ReservedClauseStartKeyword: Rule0 =
+    keyword("CALL") |
+    keyword("CREATE") |
+    keyword("CYPHER") |
+    keyword("DELETE") |
+    keyword("DO") |
+    keyword("DROP") |
+    keyword("EXPLAIN") |
+    keyword("FOREACH") |
+    keyword("FROM") |
+    keyword("INTO") |
+    keyword("LOAD") |
+    keyword("MATCH") |
+    keyword("MERGE") |
+    keyword("OPTIONAL") |
+    keyword("PERSIST") |
+    keyword("PROFILE") |
+    keyword("_PRAGMA") |
+    keyword("RELOCATE") |
+    keyword("RETURN") |
+    keyword("SNAPSHOT") |
+    keyword("START") |
+    keyword("UNION") |
+    keyword("USING") |
+    keyword("UNWIND") |
+    keyword("WITH")
```

Comment:
Nice! This is a step in the right direction in general I think. See f.e. https://github.com/opencypher/openCypher/blob/master/cip/1.accepted/CIP2016-12-19-Reserved-keywords.adoc